### DISCUSSION TITLE
Castle screen, creature recruit window, creature information window with right click, text alignment

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10311] [General]
In fheroes2, the text of the information is aligned with the colon ":".

I'm not a big fan of this presentation.
This does not make the information more readable because it is aligned with the colon ":" in fheroes2. :)
It's ugly.
I suggest displaying the information like in the original game.

In original game:

![Recruit creature info creature right H2 01](https://user-images.githubusercontent.com/43583994/93664679-05932800-fa71-11ea-92e9-b272a585aa9a.png)

In fheroes2:

![Recruit creature info creature right fh2 01](https://user-images.githubusercontent.com/43583994/93664684-0b890900-fa71-11ea-8c34-80d6473bed4a.png)

**Comment by Branikolog** [comment]
To my mind this makes information more easy to recognize...

> It's ugly.

Well... The original displaying seemes to me a bit blended...

**Comment by LeHerosInconnu** [comment]
> To my mind this makes information more easy to recognize...
> 
> > It's ugly.
> 
> Well... The original displaying seemes to me a bit blended...

I really prefer "blended" but aesthetic rather than "ugly" and ugly. :D

**Comment by ihhub** [comment]
We haven't found a consensus for this issue. Moving it to the Discussion section.
